release.yaml

```diff
@@ -1,10 +1,14 @@
-fleet-agent:
-  - 104.0.1+up0.10.1-rc.1
 fleet:
   - 104.0.1+up0.10.1-rc.1
+fleet-agent:
```

Comment:
We should avoid adding references to charts that weren't updated in this PR. Remove the `fleet-agent` and `rancher-vsphere-csi` fields please. Otherwise LGTM!

Reply:
Well, those changes are based on `make check-release-yaml`.